packages/syft/src/syft/core/smpc/store/crypto_store.py

```diff
@@ -0,0 +1,71 @@
+"""CryptoStore manages the needed crypto primitives."""
+
+# stdlib
+from typing import Any
+from typing import Callable
+from typing import Dict
+from typing import Iterable
+from typing import List
+
+# relative
+from ...common.serde.serializable import serializable
+
+
+@serializable(recursive_serde=True)
+class CryptoStore:
+    """Manages items needed for MPC Computation.
+
+    Attributes:
+        store (Dict[Any, Any]): Object needed for MPC.
+
+    """
+
+    __slots__ = ["store"]
```

Comment:
Q: I think it is better if those were tuples?

Reply:
Yeah , except `dictionary` we could have list or tuples, I would change to tuples :+1: